content/zh-cn/docs/concepts/containers/runtime-class.md

```diff
@@ -97,7 +97,7 @@ handler 必须是有效的 [DNS 标签名](/zh-cn/docs/concepts/overview/working
 <!--
 ### 2. Create the corresponding RuntimeClass resources
 
-The configurations setup in step 1 should each have an associated `handler` name, which identifies
+The configurations set up in step 1 should each have an associated `handler` name, which identifies
```

Comment:
```suggestion
The configurations setup in step 1 should each have an associated `handler` name, which identifies
```

Reply:
Very good point... I'm removing the changes to languages that are not `/en/`